vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -284,17 +296,54 @@ def parse_container_definition(container_def, pod_id)
 
     def parse_container(container, pod_id)
       {
-        :type          => 'ContainerKubernetes',
-        :ems_ref       => "#{pod_id}_#{container.name}_#{container.image}",
-        :name          => container.name,
-        :image         => container.image,
-        :restart_count => container.restartCount,
-        :backing_ref   => container.containerID,
-        :image_ref     => container.imageID
+        :type            => 'ContainerKubernetes',
+        :ems_ref         => "#{pod_id}_#{container.name}_#{container.image}",
+        :name            => container.name,
+        :image           => container.image,
+        :restart_count   => container.restartCount,
+        :backing_ref     => container.containerID,
+        :image_ref       => container.imageID,
+        :container_image => parse_container_image(container.image, container.imageID)
       }
+
       # TODO, state
     end
 
+    def parse_container_image(image, imageID)
```

Comment:
Can you add spec tests for this method? Using all the known image formats. Thanks!